db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20221114190902) do
```

Comment:
@lucacug should I manually reset this to the previous version? I've change two migrations into rake tasks.

Reply:
if you rollback the migration this should be automagically updated to the latest migration run. If you already deleted the migration (looks like so) you will have to manually update this to be the timestamp of the latest migration in the branch yeah